### PR TITLE
Add hocr-example with current Ocropus

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: samples/hocr/1.1/433934212_0017.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<head>
+<title>OCR Results</title>
+<meta http-equiv="content-type" content="text/html; charset=utf-8" />
+<meta name="Description" content="OCRopus Output" />
+<meta name="ocr-system" content="ocropus-0.4" />
+<meta name="ocr-capabilities" content="ocr_line ocr_page" />
+</head>
+<body>
+
+<div class='ocr_page' title='file 433934212_0017.bin.png'>
```
Comment (zuphilip) [inline samples/hocr/1.1/433934212_0017.html:14]: Ocropus does not output a `bbox` here.

File: samples/hocr/1.1/433934212_0017.html.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+engine: ocropy 1.0
+date: 2016-09-13
+engine-latest-commit: https://github.com/tmbdev/ocropy/tree/97ace5581b4c4deb95cfcd17b595b0ff1da36c25
```
Comment (zuphilip) [inline samples/hocr/1.1/433934212_0017.html.yml]: I don't know if such additional metadata are allowed. It should help to describe the used Ocropus version...

File: samples/hocr/1.1/433934212_0017.html.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+view-url: http://digi.bib.uni-mannheim.de/seitenansicht_suche/?id=18&tx_dlf%5Bid%5D=1448&tx_dlf%5Bpage%5D=17
+same-as:
+  - samples/image/433934212_0017.bin.png
```
Comment (zuphilip) [inline samples/hocr/1.1/433934212_0017.html.yml:4]: Well, the jpg and the binarized and normalized picture are not anymore the same. But one is derived from the other...

**Comment by zuphilip** [comment]
You can make the necessary changes in the metadata. Feel free to update this PR by using the new "Allow edits from maintainers." option.

File: samples/hocr/1.1/433934212_0017.html.yml (hunk 2 above)
Comment (kba) [reply samples/hocr/1.1/433934212_0017.html.yml]: Sure, that's useful. But maybe combine it with `engine`, i.e. use the sha as the version?

File: samples/hocr/1.1/433934212_0017.html.yml (hunk 3 above)
Comment (kba) [reply samples/hocr/1.1/433934212_0017.html.yml:4]: `same-as` is just for grouping related resources, no semantics intended.

File: samples/hocr/1.1/433934212_0017.html (hunk 1 above)
Comment (kba) [reply samples/hocr/1.1/433934212_0017.html:14]: True but then Ocropus doesn't adhere to the spec in in other places:

```
$ hocr-spec samples/hocr/1.1/433934212_0017.html
[ERROR] samples/hocr/1.1/433934212_0017.html:14 Error parsing properties for "<div class="ocr_page" title="file 433934212_0017.bin.png">" : (property class HocrSpecProperties has no attribute 'file' (AttributeError on "file"))
```

**Comment by zuphilip** [comment]
Please have a look at the new version.

**Pull request merged**